theme-kit/api.ts

```diff
@@ -0,0 +1,22 @@
+import { createPrezlyClient } from '@prezly/sdk';
+
+import { assertServerEnv } from '@/utils';
+
+import { env } from './env';
+import { createContentDeliveryClient } from './lib';
+
+export function api() {
+    assertServerEnv('api');
+
+    const { PREZLY_ACCESS_TOKEN, PREZLY_NEWSROOM_UUID, PREZLY_API_BASEURL } = env();
+
+    const client = createPrezlyClient({
+        accessToken: PREZLY_ACCESS_TOKEN,
+        baseUrl: PREZLY_API_BASEURL,
+    });
+
+    return {
+        api: client,
+        contentDelivery: createContentDeliveryClient(client, PREZLY_NEWSROOM_UUID),
+    };
+}
```

Comment:
This is a helper function that can be used anywhere during the ServerComponent rendering phase, it relies on the `env()` helper to acquire the environment variables